File: LeetCodeProblemsGo/LeetCode2011FinalValueOfVariableAfterPerformingOperations/LeetCode2011FinalValueOfVariableAfterPerformingOperations.go
package LeetCode2011FinalValueOfVariableAfterPerformingOperations

func finalValueAfterOperations(operations []string) int {
	result := 0
	for _, operation := range operations {
		for _, r := range operation {
			if r == '+' {
				result++
				break
			}
			if r == '-' {
				result--
				break
			}
		}
	}
	return result
}

File: LeetCodeProblemsCpp/LeetCodeProblemsCpp/Problems/LeetCode2011FinalValueOfVariableAfterPerformingOperations.cpp
#include "LeetCode2011FinalValueOfVariableAfterPerformingOperations.h"

int LeetCode2011FinalValueOfVariableAfterPerformingOperations::finalValueAfterOperations(std::vector<std::string>& operations)
{
    int result{};
    for (auto & operation : operations)
    {
        for (auto c:operation)
        {
            if (c == '+')
            {
                result++;
                break;
            }
            if (c == '-')
            {
                result--;
                break;
            }
        }
    }
    return result;
}
